stretch_core/launch/stretch_robot.launch.py

```diff
@@ -0,0 +1,41 @@
+from ament_index_python.packages import get_package_share_directory
+
+from launch import LaunchDescription
+from launch.actions import DeclareLaunchArgument, IncludeLaunchDescription
+from launch.conditions import IfCondition, UnlessCondition
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+from launch.substitutions import LaunchConfiguration
+
+
+def generate_launch_description():
+    ld = LaunchDescription()
+    package_prefix = get_package_share_directory('stretch_core')
+
+    driver_py = PythonLaunchDescriptionSource([package_prefix, '/launch/stretch_driver.launch.py'])
+    ld.add_action(IncludeLaunchDescription(driver_py, launch_arguments={'broadcast_odom_tf': 'true'}.items()))
+
+    resolution_arg = DeclareLaunchArgument(
+        'high_res',
+        default_value='False',
+        description='Whether to use high resolution realsense'
+    )
+
+    hi_realsense_py = PythonLaunchDescriptionSource([package_prefix, '/launch/d435i_high_resolution.launch.py'])
+    hi_realsense_launch = IncludeLaunchDescription(hi_realsense_py,
+                                                   condition=IfCondition(LaunchConfiguration('high_res')))
+
+    lo_realsense_py = PythonLaunchDescriptionSource([package_prefix, '/launch/d435i_low_resolution.launch.py'])
+    lo_realsense_launch = IncludeLaunchDescription(lo_realsense_py,
+                                                   condition=UnlessCondition(LaunchConfiguration('high_res')))
+    ld.add_action(resolution_arg)
+    ld.add_action(hi_realsense_launch)
+    ld.add_action(lo_realsense_launch)
+
+    rplidar_py = PythonLaunchDescriptionSource([package_prefix, '/launch/rplidar.launch.py'])
+    rplidar_launch = IncludeLaunchDescription(rplidar_py)
+    ld.add_action(rplidar_launch)
```

Comment:
This looks good though I think we should agree on a launch file style for ROS2. I usually (to this point) wrote these similar to moveit2 (see https://github.com/ros-planning/moveit2/blob/main/moveit_demo_nodes/run_move_group/launch/run_move_group.launch.py):

In a style where we define includes and nodes, then return them as Launch Description.

This is also fine for me I don't have a specific preference. But keeping them in one style would be better. The style choice is your call.

Reply:
First off, I am just learning to write these wacky ROS2 launch files, so I might be way off base. 

Are you talking about the difference between doing `ld.add_action` throughout and then returning the `ld` at the end vs. creating the `LaunchDescription` at the very end with a list of nodes like in the linked launch file?

Reply:
Yep, that was what I am talking about.